index.html

```diff
@@ -6,15 +6,16 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Asap:400,400i,500,500i,600,600i,700,700i&display=swap" rel="stylesheet">
     <title>Document</title>
 </head>
 <body>
 
 <header class="header" id="header-anchor">
 
-    <nav>
+    <nav class="header__navigation">
         <div class="header__left-side">
-            <img src="images/Logotype.png" alt="" class="header__logotype__image">
+            <a href="#"><img src="images/Logotype.png" alt="" class="header__logotype__image"></a>
```

Comment:
`images/logotype.png` is not a good practice. 
If your site will grow - you need to have a good hierarchy, like this: `images/header/logotype.png`